sources/api/netdog/src/main.rs

```diff
@@ -84,7 +84,6 @@ fn run() -> cli::Result<()> {
         SubCommand::GenerateHostname(_) => cli::generate_hostname::run()?,
         SubCommand::GenerateNetConfig(_) => cli::generate_net_config::run()?,
         SubCommand::SetHostname(args) => cli::set_hostname::run(args)?,
-        SubCommand::PreparePrimaryInterface(_) => cli::prepare_primary_interface::run()?,
```

Comment:
Adding here since GitHub doesn't let me too far away from the diff: The comment at the top and netdog's readme still mention the `prepare-primary-interface` command.

Reply:
Good eyes - I'll fix that up!